buddyboss/buddyboss-platform
# PROD-867 feed link preview feature improved

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2036  .

### How to test the changes in this Pull Request:

1. Add shopify or Squarespace site link in the create activity area
2. See it shows the link preview

### Proof Screenshots or Video

https://www.loom.com/share/3680a7378d124cddb009f0e140a5596e

### Other information:




### Changelog entry

> Activity feed link preview feature improved
